modules/release-notes/snip-bug-fix-rhidp-5120.adoc

```diff
@@ -0,0 +1,10 @@
+[id="bug-fix-rhidp-5120"]
+= Fixed the techdoc text selection during reporting a techdoc issue
+
+Before this update, the feature to report a documentation (techdoc) issue failed. Therefore, when a user selected a text in a techdoc, a large icon appeared instead of a tooltip-button.
```

Comment:
```suggestion
Previously, the feature to report a documentation (TechDoc) issue failed. Therefore, when a user selected a text in a TechDoc, a large icon appeared instead of a tooltip button.
```